src/main/java/ChatReques11.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques11 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
+    public boolean newMethod(String st1, String st2) {
```

Comment:
Comparison of strings using '==' instead of equals() method, which checks for content equality (Severity: Major)